azure-pipelines.yml

```diff
@@ -7,8 +7,11 @@ trigger:
 pr: none
 
 variables:
-  imageName: 'apply-for-postgraduate-teacher-training'
-  debug: true
+- group: Docker Shared variables
+- name: imageName
+  value: 'apply-for-postgraduate-teacher-training'
+- name: debug
```

Comment:
is this an intentional change?

Reply:
@tpinney you replied this was part of the work in progress — should it still be here?